### PR TITLE
[fix] minor typos in docs/admin/settings.rst

### PR DESCRIPTION
**Pull request by stavares843**
fix typo

### CODE REVIEW

**Comment by return42** [comment]
Thanks for your patch, for the future: please use meaningful commit messages ["Update settings.rst"](https://github.com/asciimoo/searx/pull/2005/commits/62a8eb401e376577334a85e4fd1b5bc69d8319fe) does not say anything.  This typical happens when you use github "edit this file" button and do not check the commit message below. Please read [contribution guide](https://asciimoo.github.io/searx/dev/contribution_guide.html) first / thanks!

**Commit messages are an ongoing document of the codebase!**


By example how meaningful a commit messages could be: if you need new passwords grab for [remove password](https://github.com/search?q=%22remove+password%22&type=Commits).

**Comment by stavares843** [comment]
was just a misclick, thanks!

**Comment by return42** [comment]
What does the [commit noise](https://github.com/asciimoo/searx/pull/2005/commits) help?

File: docs/admin/settings.rst [hunk 1]
```diff
@@ -93,7 +93,7 @@ Global Settings
   Allow your instance of searx of being able to proxy images.  Uses memory space.
 
 ``default_locale`` :
-  Aearx interface language.  If blank, the locale is detected by using the
```
Comment (kvch) [inline docs/admin/settings.rst:96]: :rofl:

**Pull request closed without merging**